holoviews/plotting/bokeh/callbacks.py

```diff
@@ -87,6 +87,9 @@ class Callback:
     # The plotting handle(s) to attach the JS callback on
     models = []
 
+    # Additional handles to hash on for uniqueness
+    extra_handles = []
```

Comment:
We need to tell the callback to hash itself based on all the handles so we don't reuse a callback attached to different axes. It's a bit ugly as you can see below.